add.go

```diff
@@ -324,7 +324,7 @@ func (b *Builder) addHelper(excludes *fileutils.PatternMatcher, extract bool, de
 					}
 					// Skip the whole directory if the pattern matches exclusively
```

Comment:
The comment does not match anymore as we are now recursing into the directory.

Reply:
Fixed